content/en/docs/client-options.md

```diff
@@ -185,6 +185,9 @@ These clients are compatible with our [staging endpoint for ACME v2](https://com
 
 # Libraries
 
+## Delphi
+- [LetsEncryptDelphi](https://github.com/tothpaul/LetsEncryptDelphi)
```

Comment:
Hi @tothpaul, thanks for working on an ACME client and submitting a PR.

Your client's name does not meet our [trademark policy](https://letsencrypt.org/trademarks/) and I unfortunately can't accept this PR as-is.

Could you rename your project? I would suggest "ACMEDelphi" or something similar. This has the added advantage of emphasizing that your project would be compatible with other ACME-based CAs.

Thanks!

Reply:
(Could you also please update the last modified date-stamp at the top of this markdown file?)

Reply:
Thanks @tothpaul - much appreciated